vividus-plugin-web-app/build.gradle

```diff
@@ -29,7 +29,8 @@ dependencies {
     implementation(group: 'org.seleniumhq.selenium', name: 'selenium-safari-driver', version: seleniumVersion)
     implementation(group: 'junit', name: 'junit', version: versions.junit4)
     implementation(group: 'io.github.bonigarcia', name: 'webdrivermanager', version: '3.6.2')
-    implementation(group: 'net.lightbody.bmp', name: 'browsermob-core', version: '2.1.5')
+    implementation(group: 'com.browserup', name: 'browserup-proxy-core', version: '2.0.0')
```

Comment:
`2.0.1` ?

Reply:
Yes, I set it on Friday, before 2.0.1 release. Will fixed